ci-operator/step-registry/openshift/upgrade/qe/test/chainupgrade/openshift-upgrade-qe-test-chainupgrade-chain.yaml

```diff
@@ -0,0 +1,10 @@
+chain:
+  as: openshift-upgrade-qe-test-chainupgrade
+  steps:
+  - ref: cucushift-upgrade-drop-last-hop
+  - ref: cucushift-chainupgrade-toimage
+  - chain: openshift-upgrade-qe-test
+  documentation: |-
+    The chain executes upgrade-prepare, which creates test data on a cluster;
+    then conduct paused upgrade;
```

Comment:
it looks like the description should be updated here.

Reply:
updated.